test/services/actionService.test.js

```diff
@@ -0,0 +1,18 @@
+
+const actionsService = require('../../server/services/actionsService')
+describe('actionsService', () => {
+  test('actions service returns a actions object containing an array', async () => {
+    expect.assertions(1)
```

Comment:
This `expect.assertions(1)` and the one below don't seem necessary and make the tests more brittle. The typical use-case for this is when we put assertions within a callback to check that the correct arguments are given to the callback. In that case, the `expect.assertions()` would cause the test to fail if the callback is never reached.

Neither this or the next test has a callback. Is there something I'm missing that means we need to test the assertion count?

Reply:
Just my paranoia about async code, removed.